meta-ivi/recipes-extended/dlt-daemon/dlt-daemon_2.18.1.bb

```diff
@@ -12,26 +12,25 @@ SECTION = "console/utils"
 LICENSE = "MPLv2"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=8184208060df880fe3137b93eb88aeea"
 
-PR = "r2"
-
 DEPENDS = "dbus zlib gzip-native"
 
-SRCREV = "bdde2374efd567f27d7a4ec70911b34cda061d27"
-SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=https \
+SRC_URI = "https://github.com/GENIVI/dlt-daemon/archive/v${PV}.tar.gz \
     file://0002-Don-t-execute-processes-as-a-specific-user.patch \
     file://0004-Modify-systemd-config-directory.patch \
-    file://0001-Fix-compilation-with-glibc-2.28.patch \
-    "
-S = "${WORKDIR}/git"
+"
+SRC_URI[md5sum] = "6f2f6d3386991ed5d5e5078647c62ce5"
+SRC_URI[sha256sum] = "07f7a65d3d248e8f5eed820cae0e2ec81318417a81540647346f81cfe024ee65"
+
+S = "${WORKDIR}/${PN}-${PV}"
 
 inherit autotools gettext cmake systemd
 
 PACKAGES += "${PN}-systemd"
 SYSTEMD_PACKAGES = "${PN} ${PN}-systemd"
 SYSTEMD_SERVICE_${PN} = "dlt-system.service dlt.service"
 SYSTEMD_AUTO_ENABLE_${PN} = "enable"
-SYSTEMD_SERVICE_${PN}-systemd = "dlt-example-user.service \
-    dlt-dbus.service \
+SYSTEMD_SERVICE_${PN}-systemd = " \
+    dlt-example-user.service \
```

Comment:
So this means the DLT systemd service will be called dlt-example-user.service in a running system? We do not want that?

Reply:
Nm, I see now that this is only for the example and dlt.service is still added above. This is only example services?

Reply:
seems to be: https://github.com/GENIVI/dlt-daemon/blob/master/systemd/dlt-example-user.service.cmake